game/src/game/player/mod.rs

```diff
@@ -1208,43 +1221,87 @@ fn player_update_passive_buffs(
     mut query: Query<(
         &Passives,
         &LinearVelocity,
+        &GlobalTransform,
         &EnemiesNearby,
         &BuffTick,
         &mut PlayerStatMod,
+        Option<&Grounded>,
```

Comment:
```suggestion
        Has<Grounded>,
```

Reply:
you can use Has<Grounded> here and then check the bool later instead of needing grounded.is_some()